### PR TITLE
Show the worker UUID in machines/tasks tables

### PR DESCRIPTION
**Pull request by ppigazzini**
Show only the first UUID group of hex digits

### CODE REVIEW

**Comment by ppigazzini** [comment]
@tomtor please review this PR at your convenience, I'm not very confident with my knowledge of the server logic :)
DEV updated, it seems working fine also for very old tests.

File: server/fishtest/util.py [hunk 1]
```diff
@@ -33,6 +33,8 @@ def get_worker_key(task):
     suffix = UUID_MAP[username][uuid]
     if suffix != 0:
         worker_key += "-" + str(suffix)
+    if len(uuid) != 0:
```
Comment (tomtor) [inline server/fishtest/util.py:36]: Is len(uuid) ever 0? Can this condition be removed?

**Review by tomtor — COMMENTED** [review]
Looks good to me, but see comment

File: server/fishtest/util.py (hunk 1 above)
Comment (ppigazzini) [reply server/fishtest/util.py:36]: `uuid=""` for a very old test that lacks `"unique_key"`:
https://github.com/glinscott/fishtest/blob/55f4f9f66d618c6f627aec4222acb116cecb3b27/server/fishtest/util.py#L26

See this test on DEV:
https://dfts-0.pigazzini.it/tests/view/5134df478f0c3e3c12c7465c

Comment (tomtor) [reply server/fishtest/util.py:36]: Ah, clear.

**Review by tomtor — APPROVED** [review]
Nice!

**Pull request merged**